types/autoprefixer/autoprefixer-tests.ts

```diff
@@ -27,7 +27,7 @@ const deprecationTest = autoprefixer({
 });
 
 autoprefixer.info(); // $ExpectedType () => void
-autoprefixer.data; // $ExpectType { browsers: any; prefixes: any; }
+autoprefixer.data; // $ExpectType any
```

Comment:
No, it does not work (so you typed any), because `autoprefixer` in your version is untyped
(that is, is `any` type itself),
```suggestion
autoprefixer.data; // $ExpectType { browsers: any; prefixes: any; }
```